index.js

```diff
@@ -216,3 +216,8 @@ function initCustomServiceEnvironment(envSchema = defaultSchema) {
 }
 
 module.exports = initCustomServiceEnvironment
+module.exports.getDirectServiceProxy = getDirectlyServiceBuilderFromService
+module.exports.getServiceProxy = (microserviceGatewayServiceName) => {
+  this.MICROSERVICE_GATEWAY_SERVICE_NAME = microserviceGatewayServiceName
+  return getServiceBuilderFromService.bind(this)
+}
```

Comment:
```suggestion
module.exports.getServiceProxy = function (microserviceGatewayServiceName, baseOptions = {}) {
  return getServiceBuilderFromService(MICROSERVICE_GATEWAY_SERVICE_NAME, {}, baseOptions)
}
```

In this way we avoid to change `this` and the double function invocation

Reply:
to do this we should change the `getServiceBuilderFromService(baseOptions = {})` signature or we can directly call the `serviceBuilder(MICROSERVICE_GATEWAY_SERVICE_NAME, {}, baseOptions)`

Reply:
right, maybe the second way is better